### ISSUE TITLE
savefig output blank eps files.

### ISSUE THREAD

**Issue by lhcamilo** [matplotlib/matplotlib#5190]
My scripts had been working fine before, but suddently I realised that all the eps files that I generated were blank.

I am using matplotlib 1.4.3  on two machines running python 3.4.3 and 3.5.0 on ubuntu 15.04. Despite that, I am able to output .pdf  plots just fine. The problem seems to be confined to .eps files.

**Comment by tacaswell** [comment]
Can you provide a minimal example which outputs and empty eps, but works with pdf?  With out any code it is _very_ hard to help you.

**Comment by lhcamilo** [comment]
Thank for your reply I have just tested it with this simple code below that I used to test some settings. Unfortunately the code still produces blank .eps but pdfs work fine 

In one of my machines , I also tried setting `ps.usedistiller: xpdf` but that had no effect.

```
import numpy as np
import matplotlib.pyplot as plt


xData =  np.linspace(0,2*np.pi,100,endpoint=True)
yData1 = np.sin(xData)



plt.rc('text', usetex=False)
plt.rc('font', family='serif', size=18)
plt.rc('lines', linewidth=2, antialiased=True)
plt.rc('legend',fontsize=14, frameon=False)


plt.figure(num=1, figsize=(8, 6))
plt.ylabel(r'sin$(\theta)$')
plt.xlabel(r'$\theta$')


plt.plot(xData, yData1, color='k', linestyle='-', label='y1 data')
plt.xlim(0,2*np.pi)

plt.xticks([0, np.pi/2, np.pi, 3*np.pi/2, 2*np.pi],
           ['$0$', r'$\frac{\pi}{2}$', r'$\pi$', r'$\frac{3\pi}{2}$', r'$2\pi$'],size=18)

plt.legend(loc=0)

plt.savefig('sine.eps', format='eps', dpi=600)
plt.savefig('sine.pdf', format='pdf', dpi=600)

```

**Comment by mdboom** [comment]
I'm not able to reproduce.  Do you have a backend set in ~/.config/matplotlib/matplotlibrc?

**Comment by lhcamilo** [comment]
Nope, my ~/.config/matplotlib/ is blank.

**Comment by mdboom** [comment]
Can you share one of the broken eps files?

**Comment by lhcamilo** [comment]
Right... now I feel a bit silly. As it turns out I seem to have found the root of the problem. The two machines in which I have been producing plots were running ubuntu 15.04 , one of them is a (jupyter) server and another is laptop. So I have been accessing the server from the laptop and also running things locally. 

Now that I am at my desktop workstation (running RHEL) I have just downloaded one of .eps files that I produced... and it looks fine. So the only think I can conclude from this is that the problem is not that matplotlib is having problems producing eps files, but that my laptop is having issues reading them (with both okular and evince).

I am sorry for wasting your time but thanks for your help anyway. 

I am now closing this bug,
